src/DropDownMenu/DropDownMenu.js

```diff
@@ -111,6 +111,10 @@ class DropDownMenu extends Component {
      */
     disabled: PropTypes.bool,
     /**
+     * Overrides default `SvgIcon` dropdown arrow component.
+     */
+    iconButtonElement: PropTypes.node,
```

Comment:
What about the following name `iconButton`? I don't think that `Element` is adding any value.